swarmcg/evaluate_model.py

```diff
@@ -191,14 +213,17 @@ def main():
 								action='store_true', default=False)
 
 	# display help if script was called without arguments
-	if len(sys.argv) == 1:
-		args_parser.print_help()
-		sys.exit()
+	# if len(sys.argv) == 1:
+	# 	args_parser.print_help()
+	# 	sys.exit()
```

Comment:
Let's no do this in fact, because we might want to execute the script from within a directory in which all files are already present